src/Orleans.Core/Core/ClientBuilderExtensions.cs

```diff
@@ -19,7 +22,7 @@ public static class ClientBuilderExtensions
         /// <param name="builder">The host builder.</param>
         /// <param name="siloName">The silo name.</param>
         /// <returns>The silo builder.</returns>
-        public static IClientBuilder ConfigureDefaults(this IClientBuilder builder)
+        public static IClientBuilder Configure(this IClientBuilder builder)
```

Comment:
Isn't this a convenience function to setup common defaults for the client?  If so, didn't the previous name communicate that better?

Reply:
The SiloHostBuilder has a method called `Configure` which does the same thing as this. In that case, the method was previously called `ConfigureOrleans`.

I'm happy to revert these changes

Reply:
no matter what we choose, the naming need to be consistent with silo and client. +1 for configureDefaults